akka-cluster-sharding-typed/src/main/scala/akka/cluster/sharding/typed/internal/ClusterShardingImpl.scala

```diff
@@ -150,10 +158,10 @@ import akka.japi.function.{ Function ⇒ JFunction }
         untypedSharding.internalStart(
```

Comment:
this will happily return the previous ref for the "typeKey.name" if it was already registered